arduino/smartcar/smartcar.ino

```diff
@@ -27,8 +27,9 @@ void setup()
 void loop()
 {
   const auto distance = front.getDistance();
-  // The car comes to a stop if the Front UltraSonic reads a distance of 1.5m or lower
-  if (distance > 0 && distance < 1500) {
+  // The car comes to a stop if the Front UltraSonic reads a distance of 1 metre or lower.
+  // The distance upper bound in the if block is slightly greater than 1 metre, however, as the car overcomes its inertia during motion first before coming to a halt.
+  if (distance > 0 && distance < 150) {
```

Comment:
As I mentioned in the other pull request you, this still doesn't do what your comment right above claims